.github/workflows/ci.yml

```diff
@@ -462,6 +496,8 @@ jobs:
       fail-fast: false
       matrix: ${{ fromJson(needs.build-test-matrix.outputs.matrix) }}
     timeout-minutes: 60
+    env:
+      SECRETS_PRESENT: ${{ secrets.SECRETS_PRESENT }}
```

Comment:
why the indirection? Why not inline `secrets.SECRETS_PRESENT`?

Also if you need it everywhere you can add it to top-level `env` key probably.

Reply:
Secrets are not available in condition blocks, this is what githubs manual recommends